### ISSUE TITLE
epresent render headlines too smallish

### ISSUE THREAD

**Issue by ydroneaud** [eschulte/epresent#53]
Hi,

I'm considering using epresent to run a presentation written in org-mode:
- source code is available at https://gitorious.org/opteya/talk-kernel-userspace-abi/ 
- presentation (S5 exported ;) is available at http://opteya.com/talks/2014/kernel-recipes/lightning-talk-kernel-userspace-ABI/

Unfortunately, I have small issues to report.

This is the fourth one.

headlines are too small compared to the content

See attached 
![screenshot](https://cloud.githubusercontent.com/assets/881377/16619423/42dfc446-438e-11e6-8597-b4243d8eac5f.png)

I'm using Fedora 24, GNOME 3.20, GNU Emacs 25.0.94.1, Org-mode version 8.2.10, epresent 32e486c

**Comment by sellout** [comment]
You’re right about the various font sizes being odd. This is my fault, I think. I submitted some changes to the faces, but have others that are not yet submitted. Thankfully, these settings are customizable …

You can change the base body (and page number size) with `epresent-text-size`, and the title, etc. via the faces available via `M-x customize-group epresent`.

I will try to submit my other changes that give more control and consistency soon.

**Comment by sellout** [comment]
While I think this issue is technically fixed by #57 above, I think it would be great if the default face sizes were relative to the size of the presentation frame. I know there have been other complaints about the faces being too big, and I think making them a bit more dynamic would help (especially when it turns out the projector doesn’t have the same idea about face sizes that your monitor does).
